docs/scipp-neutron/instrument-view.ipynb

```diff
@@ -93,14 +108,13 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "![spheview](../images/neutron/instrument_view_sphe.png)"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "This works for any file that can be loaded by Mantid that contains a valid instrument description or geometry:"
+    "This works for any file that can be loaded by Mantid that contains a valid instrument description or geometry.\n",
+    "\n",
+    "There are also two different rendering modes:\n",
```

Comment:
This doesn render the list, I think you need a newline here.

Reply:
ok strange, it does render the list on my local notebook...